www/modules/migrate/controllers/google-calendar.client.controller.es6

```diff
@@ -3,16 +3,17 @@ const DATE_FORMAT = 'yyyy-MM-dd';
 
```

Comment:
Is better to store const in https://docs.angularjs.org/api/auto/service/$provide#constant